data/competencies.yml

```diff
@@ -495,9 +497,10 @@
   domain: null
 
 - id: d8651fc0-059b-4c11-929f-e07803f9f8bf
-  summary: Understands and considers other teams' needs
+  summary: Actively seeks the views of other teams to help guide work
```

Comment:
The changes to this competency are unrelated. Were these supposed to be a separate PR?

Reply:
I found and approved the separate PR. This may conflict

Reply:
Doh!